pom.xml

```diff
@@ -1191,6 +1191,18 @@
         </plugins>
       </build>
     </profile>
+    <profile>
+      <id>mac-apple-silicon</id>
+      <activation>
+        <os>
+          <family>mac</family>
+          <arch>aarch64</arch>
+        </os>
+      </activation>
+      <properties>
+        <nar.aolProperties>${project.build.directory}/../src/apple_m1_aol.properties</nar.aolProperties>
```

Comment:
Why not just `src/apple..` ?

Reply:
@nicoloboschi Great! Fixed, I don't that way before